src/macros.h

```diff
@@ -233,6 +233,8 @@
 #define MB_PTR2LEN(p)	    (has_mbyte ? (*mb_ptr2len)(p) : 1)
 /* Advance multi-byte pointer, skip over composing chars. */
 #define MB_PTR_ADV(p)	    p += has_mbyte ? (*mb_ptr2len)(p) : 1
+/* Decrease p with a character, like MB_PTR_ADV(). */
+#define MB_PTR_DECR(p)	    p -= has_mbyte ? (*mb_ptr2len)(p) : 1
```

Comment:
Oh, I was wrong.

I fixed this as below because I think this checks only one byte characters.

```c
    static int
get_repeated_single_quotes_length(char_u *x, char_u *whole_start)
{
    char_u *follower;
    char_u *following;

    if (*x != '\'')
    {
        return 0;
    }

    for (follower = x; *follower == '\''; --follower)
    {
    }
    for (following = x; *following == '\''; ++following)
    {
    }

    return (x - follower) + (following - x) + 1;  // 1 is num of *x
}
```